### PR TITLE
In contests owned

### PR DESCRIPTION
**Pull request by UVStudio**
Completed displaying Contests Owned in profile
started Stripe work

### CODE REVIEW

File: client/src/components/ContestCardPanel.js [hunk 1]
```diff
@@ -9,21 +17,29 @@ export default function ContestCardPanel(props) {
...
-            className={classes.media}
-            //image={require(`../assets/${contest.thumbnail}`)}
-            image="https://team-owl-tattoo.s3.ca-central-1.amazonaws.com/tattoos/tattoo1.png"
-            title="hi"
-            height="140"
-          />
+          {contest.contestPics ? (
+            <CardMedia
+              className={classes.media}
+              //image={require(`../assets/${contest.thumbnail}`)}
```
Comment (bonnieli) [inline client/src/components/ContestCardPanel.js]: delete if not needed anymore

File: client/src/components/TabPanel.js [hunk 2]
```diff
@@ -5,8 +5,11 @@ import TabPanelStyles from './TabPanelStyles';
 export default function TabPanel(props) {
   const { value, index, contests, Component, ...other } = props;
   const classes = TabPanelStyles();
-  const cardsJSX = contests.map((contest, i) => {
-    return <Component key={contest.id || 1} value={value} contest={contest} />;
+  console.log(contests);
```
Comment (bonnieli) [inline client/src/components/TabPanel.js]: remove

File: server/controllers/stripe.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
+const stripe = require('stripe')(process.env.STRIPE_SECRET_KEY);
+
+const url = 'https://localhost:3000/edit_profile';
```
Comment (bonnieli) [inline server/controllers/stripe.js:3]: you can consider storing an env var called `CLIENT_DOMAIN=https://localhost:3000` so if you ever do deploy this, you can easily control between a domain vs localhost :)

**Review by bonnieli — APPROVED** [review]

File: client/src/components/TabPanel.js (hunk 2 above)
Comment (UVStudio) [reply client/src/components/TabPanel.js]: Done!

File: client/src/components/ContestCardPanel.js (hunk 1 above)
Comment (UVStudio) [reply client/src/components/ContestCardPanel.js]: Done!

File: server/controllers/stripe.js (hunk 3 above)
Comment (UVStudio) [reply server/controllers/stripe.js:3]: Updated on my current local working branch. Will be reflected in the next PR. Thanks!

**Pull request merged**